doc/article/en-US/router-configuration.md

```diff
@@ -326,9 +326,8 @@ The function passed to `mapUnknownRoutes()` has to return:
 
         config.title = 'Aurelia';
 
-        let handleUnknownRoutes = (instruction: NavigationInstruction): {moduleId: string} => {
```

Comment:
I've updated the PR to use option number 2, which seems like a sensible solution.